CppCoreGuidelines.md

```diff
@@ -17262,7 +17264,7 @@ This `Addable` violates the mathematical rule that addition is supposed to be co
 
 ##### Note
 
-The ability to specify a meaningful semantics is a defining characteristic of a true concept, as opposed to a syntactic constraint.
+The ability to specify meaningful semantics is a defining characteristic of a true concept, as opposed to a syntactic constraint.
```

Comment:
See previous comment about semantics.

Reply:
Ok, so this one is actually different [from the other one](https://github.com/isocpp/CppCoreGuidelines/pull/1776#discussion_r615805908).

I'm fine with reverting this change.

Reply:
So... Should I just revert this change?